elements/pfe-card/src/pfe-card.story.js

```diff
@@ -0,0 +1,266 @@
+  const slots = {
+    header: {
+      title: "Header"
+    },
+    body: {
+      title: "Body"
+    }
+  };
+
+  //-- Add default content to slot objects
+
+  // Build the default header content
+  slots.header.default = defaultHeading;
+
+  // Build the default body content
+  slots.body.default = defaultBody;
+
+  // Trigger the auto generation of the knobs for slots
+  config.has = tools.autoContentKnobs(slots, storybookBridge);
+
+  // Add the footer slot manually, does not accept user input via field
+  slots.footer = {
+    title: "Footer"
+  };
+
+  // Create an object for the footer attributes
+  let footerAttrs = {};
+  let ctaText;
+  let ctaLink;
+
+  // Manually ask user if they want a CTA included
+  const ctaValue = storybookBridge.boolean(
+    "Include a call-to-action?",
+    true,
+    "Call-to-action"
+  );
+
+  // If they do, prompt them for the cta text and style
+  if (ctaValue) {
+    ctaText = storybookBridge.text("Text", "Learn more", "Call-to-action");
+    ctaLink = storybookBridge.text("Link", "#", "Call-to-action");
+    const ctaPriorityValue = storybookBridge.select(
+      "Priority",
+      {
+        default: "default",
+        primary: "primary",
+        secondary: "secondary"
+      },
+      "default",
+      "Call-to-action"
+    );
+
+    // Print the priority attribute if it's not default
+    if (ctaPriorityValue !== "default") {
+      footerAttrs.priority = ctaPriorityValue;
+    }
+
+    // If the card uses a dark theme, add the on="dark" attribute
+    if (
+      ["dark", "darkest", "accent", "complement"].includes(config.prop.color)
+    ) {
+      footerAttrs.on = "dark";
+    }
+  }
+
+  // Build the default footer component
+  slots.footer.default = "";
+
+  // If the link exists, add the default value for the footer slot
+  if (ctaValue) {
+    slots.footer.default = tools.component("pfe-cta", footerAttrs, [
+      {
+        content: `<a href="${ctaLink}">${ctaText}</a>`
+      }
+    ]);
+  }
+
+  config.slots = [
+    {
+      slot: "header",
+      content: tools.customTag({
+        tag: "h3",
+        content: config.has.header
+      })
+    },
+    {
+      content: config.has.body
+    },
+    {
+      slot: "footer",
+      content: slots.footer ? slots.footer.default : ""
+    }
+  ];
+
+  // Some attribute values don't need to be included in the markup
+  if (config.prop.color === "default") {
+    config.prop.color = "";
+  }
+
+  if (config.prop.size === "standard") {
+    config.prop.size = "";
+  }
+
+  let rendered = template(config);
+
+  cleaner.clean(
+    rendered,
+    {
+      indent: "    ",
+      "remove-attributes": [],
+      wrap: 0
+    },
+    html => (rendered = html)
+  );
+
+  return `${rendered}
+<pre style="white-space: pre-wrap; padding: 20px 50px; background-color: #f0f0f0; font-weight: bold;border: 1px solid #bccc;">
+${tools.escapeHTML(rendered)}
+</pre>
+`;
+
+  //   valueAttr =
```

Comment:
@castastrophe Do we want to remove all of these comments?

Reply:
Yes, I'll do that now

Reply:
bc9262f1e732cfa23cd8f6f82129f6a475b62d41